build/IPDK_Container/scripts/rundemo.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
+#Copyright (C) 2021 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+
+stty -echoctl # hide ctrl-c
+
+# Exit function
+exit_function()
+{
+    echo "Exiting cleanly"
+    pushd /root || exit
+    rm -f network-config-v1.yaml meta-data user-data
+    rm -rf /tmp/vhost-user-*
+    rm -f vm1.qcow2 vm2.qcow2
+    popd || exit
+}
+
+trap 'exit_function' SIGINT
+
+echo ""
+echo "Cleaning from previous run"
+echo ""
+
+rm -rf /tmp/vhost-user-*
+killall ovsdb-server
+killall ovs-vswitchd
+
+#echo ""
+#echo "Pulling down cirros image and configuring"
+#echo ""
+#
+#pushd /root || exit
+#wget -nc http://download.cirros-cloud.net/0.5.2/cirros-0.5.2-x86_64-disk.img
+#cp cirros-0.5.2-x86_64-disk.img vm1.qcow2
+#cp cirros-0.5.2-x86_64-disk.img vm2.qcow2
+#popd || exit
```

Comment:
Using cirros didn't work? Maybe its better to remove the lines then instead of commenting them out.  Now it looks like people can use cirros  instead of the ubuntu images below if they want to.

Reply:
Yes, cirrus had issues with cloud-init. I will remove, good catch.